### DISCUSSION TITLE
Подключение два контроллера к PS2

### DISCUSSION THREAD

**Discussion by Zabludilsya** [darthcloud/BlueRetro#1108] [Q&A]
Привет народ! Ребята,а как подключить 2 ( два) контроллера от PS4 к PS2Fat? Один подключается,а второй постоянно в поиске ( оба контроллера оригинальные и новые) Зажимаю кнопку BOOT для сопряжения второго контроллера секунд на 5 от PS4 и модуль выключает первый контроллер! Настройки сделаны на чипе на 2 контроллера и оба контроллера были отвязаны от других устройств.Или нельзя подключить два контроллера от PS4 одновременно? Спасибо.

**Comment by FatBaldDad** [comment]
I'm no Blue Retro expert at all, but it sounds as you are trying to pair with both controllers on at the same time.  Need more information.

If this is a Blue Retro Core and not something you built (becuase there could be problems with your build) then you need to pair one controller at a time.  Pair controller A, turn off then pair controller B.  Note that player assignment (player 1, player 2, etc.) is based on the order in which the Blue Retro Core detects the controllers when they are powered on.  So if you have a   specific controller you want to be player 1 be sure to turn on that one first and so on.

I don't know if I understand the issue you have, but I am just trying to help.
